docs/source/use_cases/index.rst

```diff
@@ -1,25 +1,96 @@
+This approach allows creating private or public applications with the possibility to securely interact between different domains.  
+
+
 Use case scenarios
 ==================
+We list some number of abstract use cases and specific advantages that Iroha can introduce to this application. We hope that the applications and use cases will inspire developers and creators to further innovation with Hyperledger Iroha.  
 
-By reviewing application areas together, we can understand what Iroha is capable of providing.
-Use cases here depict the mainstream scenarios of blockchain frameworks.
-It can also be a starting point for your imagination to come up with new and more interesting ways where Iroha can be used.
 
-(In this section we want to see something like https://github.com/takemiyamakoto/hyperledgerwp/tree/master/RelevantUseCases referring to Iroha. The idea is to describe as much as we can so that people can be inspired to use Iroha and can also use their imagination to come up with interesting application. This can be expanded later with good suggestions on hackatons and real-life examples in such use cases.)
+Certificates in Education, Healthcare 
+---------------------------- 
+
+Hyperledger Iroha incorporates into the system multiple certifying authorities such universities, schools and medical institutions. 
+The flexible permission model used in Hyperledger Iroha allows building certifying identities and grant certificates. 
+The storage of explicit and implicit information in users' account build various reputation and identity systems utilizing information from different sources and context.   
+
+Using Hyperledger Iroha each education or medical certificate can be verified that it was given by certification authorities. Immutability and clear validation rules provide transparency to health and education significantly reducing the usage of fake certificates.  
 
 Cross-border asset transfers
 ----------------------------
 
+Hyperledger Iroha provides fast and clear trade and settlement rules using multi-signature accounts and atomic exchange.   
+Asset management is easy as in centralized systems while providing necessary security guarantees.   
+ 
+By simplifying the rules and commands need to create and transfer assets, we lower the barrier to entry, while at the same time maintaining high-security guarantees.  
+
+Financial applications 
+----------------------------
+
+Hyperleger Iroha can be very useful in the auditing process. Each information in the system has defined access rules. These access rules can be defined at different levels: user-level, domain-level or system-level.  At the user-level privacy rules for a specific individual are defined. If access rules are defined determined at domain or system level, they are affecting all user in the domain. In Hyperledger Iroha we provide convenient role-based access control, where each role has specific permissions. 
```

Comment:
We don't have access rules, we have validation rules and access control with roles